src/components/Badge/Badge.story.jsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, select, text } from '@storybook/addon-knobs';
+import { generateVariantsFor } from '../../utils/propsVariants';
+import Badge from './Badge';
+import * as theme from '../../../tailwind';
+
+const stories = storiesOf('Badges', module);
+
+stories.add(
+  'Default Badge',
+  () => {
+    const variants = generateVariantsFor({
+      color: Object.keys(theme.colors),
+      loading: [true, false],
+      text: ['valid', 'invalid'],
+    });
+    return (
+      <section>
+        <Badge
+          className="hide-in-tests"
+          text={text('Badge Text', 'New')}
+          color={select('Badge Color', Object.keys(theme.colors), 'blue')}
+          loading={boolean('Loading', false)}
+        />
+        <div className="show-in-tests hidden">
```

Comment:
https://percy.io/Kids-First-DRC/KF-UIkit/builds/1730759/view/112736786/1280?mode=diff&browser=chrome&snapshot=112736786 looks like these aren't wrapping let's try adding `percy-min-width` to the show-in-tests div and see if that helps